packages/build-utils/src/edge-function.ts

```diff
@@ -39,7 +39,12 @@ export class EdgeFunction {
   assets?: { name: string; path: string }[];
 
   /** The regions where the edge function will be executed on */
-  regions?: 'auto' | string[] | 'all' | 'default';
+  regions?:
+    | 'auto'
+    | string[]
+    | 'all'
+    | 'default'
+    | (string & Record<never, unknown>);
```

Comment:
How about `string[] | string`?

Reply:
Then we won't have autocomplete when using it. It's also kinda nice that we show the different options

What do you think?

Reply:
Its already not showing autocomplete for your example: `regions: 'iad1'`

Reply:
but it is for `all` and `default` 😁

Reply:
Better to add JSDoc since its not even clear what "default" is or how to find which regions "all" goes to

Reply:
Also what happens if I do this?

```ts
{
  regions: ['auto', 'all', 'default', 'iad1']
}
```